citizen/citizen.c

```diff
@@ -401,33 +446,35 @@ int getMalePopulation(List *list) {
  * @param list A pointer to the list of citizens.
  * @return The number of female citizens in the list.
  */
-int getFemalePopulation(List *list) {
-    Node *current = list->head;   // Start from the head of the list
-    int femalePopulation = 0;     // Counter for female population
+int getfemale_population(List *list)
+{
+    Node *current = list->head;
+    int female_population = 0;     
 
-    // Traverse the list and count females
-    while (current != NULL) {
-        if (current->citizen.gender == FEMALE) {
-            femalePopulation++;
+    while (current != NULL)
+    {
+        if (current->citizen.gender == FEMALE)
+        {
+            female_population++;
         }
-        current = current->next; // Move to the next citizen in the list
+        current = current->next; 
     }
-    return femalePopulation;
+    return female_population;
 }
 
 /**
  * @brief Get the oldest citizen from the list.
  *
- * This function sorts the citizens by birthdate and returns the oldest citizen.
+ * This function sorts the citizens by birth_date and returns the oldest citizen.
  *
  * @param list A pointer to the list of citizens.
  * @return The oldest citizen in the list.
  */
-Citizen getOldestCitizen(List *list) {
-    // Sort the list by birthdate in ascending order
-    List sortedBdate = sortCitizen(list, BirthDate);
-    Citizen oldestCitizen = sortedBdate.head->citizen; // First citizen in sorted list is the oldest
-    return oldestCitizen;
+Citizen getoldest_citizen(List *list)
```

Comment:
please me consistent, the previous uses camelCase on the funtion naming convention

Reply:
Applied, thanks.